src/maggma/stores/aws.py

```diff
@@ -266,10 +266,15 @@ def update(self, docs: Union[List[Dict], Dict], key: Union[List, str, None] = No
                 search_doc["compression"] = "zlib"
                 data = zlib.compress(data)
 
+            search_docs.append(search_doc.copy())
+
+            search_doc[self.last_updated_field] = str(
+                search_doc[self.last_updated_field]
```

Comment:
use `maggma.utils.to_isoformat_ceil_ms` to convert the `datetime` object to a `str` so that the format is controlled by maggma.